pkg/querier/api/consistency_test.go

```diff
@@ -1,72 +0,0 @@
-// SPDX-License-Identifier: AGPL-3.0-only
-
-package api
-
-import (
-	"context"
-	"fmt"
-	"testing"
-
-	"google.golang.org/grpc"
-	"google.golang.org/grpc/metadata"
-)
-
-var (
-	// The following fixture has been generated looking at the actual metadata received by Grafana Mimir.
-	exampleIncomingMetadata = metadata.New(map[string]string{
-		"authority":            "1.1.1.1",
-		"content-type":         "application/grpc",
-		"grpc-accept-encoding": "snappy,gzip",
-		"uber-trace-id":        "xxx",
-		"user-agent":           "grpc-go/1.61.1",
-		"x-scope-orgid":        "user-1",
-	})
-)
-
-func BenchmarkReadConsistencyServerUnaryInterceptor(b *testing.B) {
-	for _, withReadConsistency := range []bool{true, false} {
-		b.Run(fmt.Sprintf("with read consistency: %t", withReadConsistency), func(b *testing.B) {
-			md := exampleIncomingMetadata
-			if withReadConsistency {
-				md = metadata.Join(md, metadata.New(map[string]string{consistencyLevelGrpcMdKey: ReadConsistencyStrong}))
-			}
-
-			ctx := metadata.NewIncomingContext(context.Background(), md)
-
-			for n := 0; n < b.N; n++ {
-				_, _ = ReadConsistencyServerUnaryInterceptor(ctx, nil, nil, func(context.Context, any) (any, error) {
-					return nil, nil
-				})
-			}
-		})
-	}
-}
-
-func BenchmarkReadConsistencyServerStreamInterceptor(b *testing.B) {
```

Comment:
is it difficult to get these benchmarks into the new implementation? I guess we can write them again if we need them, but from afar they look like they can be copied without much overhead